docs/configure-kibana-endpoint.asciidoc

```diff
@@ -102,3 +105,15 @@ apm-server.kibana.ssl.key: "/etc/pki/client/cert.key
 
 For information on the additional SSL configuration options,
 see <<configuration-ssl>>.
+
+[float]
+=== Agent Config configuration options
+
+You can specify the following options in the `apm-server.agent.config` section of the
++{beatname_lc}.yml+ config file:
+
+[float]
+==== `agent.config.cache.expiration`
+
+When using APM Agent configuration, information fetched from Kibana will be cached in memory.
+This setting specifies the time before cache key expiration. Defaults to 30 seconds.
```

Comment:
@graphaelli @simitt I don't know if we should document what this is used for:
- if we don't, users might think that higher value is better and cause a negative effect (imagine them setting it to 1 day)

- if we do, then we are documenting agent behaviour in the server and dictating how agents should behave:
The goal with Cache-Control was to *enable* agents to use it at their will or ignore it (the same as etag) - it was simply an attempt to return an appropriate response and provide mechanisms for optimization, should they want it.
Agents decided to use it as a poll interval for now. If we document that here, then we are forcing them to keep that behaviour, which was never my intention. They will not be able to replace their logic without introducing a breaking change.